### PR TITLE
fixed bug for M2 GGA alleles with zero coverage

### PR DESCRIPTION
**Pull request by davidbenjamin**
@takutosato here's an edge case that showed up in MC3.

### CODE REVIEW

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java [hunk 1]
```diff
@@ -332,6 +332,27 @@ public void testGivenAllelesMode() throws Exception {
...
+        final List<String> args = Arrays.asList("-I", bam.getAbsolutePath(),
+                "-" + M2ArgumentCollection.TUMOR_SAMPLE_SHORT_NAME, sample,
+                "-R", b37_reference_20_21,
+                "-L", "20:1119000-1120000",
+                "-O", unfilteredVcf.getAbsolutePath(),
+                "--genotyping-mode", "GENOTYPE_GIVEN_ALLELES",
+                "--alleles", givenAllelesVcf.getAbsolutePath());
+        runCommandLine(args);
+
+        int j = 5;
```
Comment (takutosato) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java]: remove this stray debugging line

**Comment by davidbenjamin** [comment]
@madduran This should fix the bug you saw.
